cmd/backup/encrypt_archive.go

```diff
@@ -38,14 +91,11 @@ func (s *script) encryptArchive() error {
 	}
 	defer outFile.Close()
 
-	_, name := path.Split(s.file)
-	dst, err := openpgp.SymmetricallyEncrypt(outFile, []byte(s.c.GpgPassphrase), &openpgp.FileHints{
-		FileName: name,
-	}, nil)
+	dst, closeCallback, err := encrypt(outFile)
 	if err != nil {
 		return errwrap.Wrap(err, "error encrypting backup file")
 	}
-	defer dst.Close()
+	defer closeCallback()
```

Comment:
This will now also need its error being handled. Check https://trstringer.com/golang-deferred-function-error-handling/ for an explanation on how to do this for a deferred function call.

Reply:
should line 92, 104 also be handled

Reply:
Ideally, yes. I think the linter does not complain here as it's not part of the changeset. If you still want to add this, it'd be great.

Reply:
No problem, I looked through this repo and its denfiently something that needs to be done in other files as well. I will do it in `encrypt_archive.go`

Reply:
fixed in [007b0eb](https://github.com/offen/docker-volume-backup/pull/456/commits/007b0eb5b222eb148b1d512a358cebb6a5516a93)